recipes/sokol/all/conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, tools
+import os
+import glob
+
+
+class SokolConan(ConanFile):
+    name = "sokol"
+    description = "Simple STB-style cross-platform libraries for C and C++, written in C."
+    topics = ("sokol", "graphics", "3d")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/floooh/sokol"
+    license = "ZLIB"
```

Comment:
```suggestion
    license = "Zlib"
```

https://spdx.org/licenses/Zlib.html